Tests/ApolloTests/HTTPTransportTests.swift

```diff
@@ -193,6 +196,50 @@ class HTTPTransportTests: XCTestCase {
                                                      delegate: self)
     XCTAssertNotEqual(self.networkTransport, nonIdenticalTransport)
   }
+
+  func testErrorDelegateWithErrors() throws {
+    self.graphQlErrors = []
+    let query = HeroNameQuery()
+    let body = ["errors": [["message": "Test graphql error"]]]
```

Comment:
Because `GraphQLError` is not encodable or NSObject we can not use it. 
Maybe we should consider to make `GraphQLError` encodable?